src/analysis/special_functions/complex/circle.lean

```diff
@@ -86,3 +86,10 @@ periodic_exp_map_circle.sub_eq x
 
 lemma exp_map_circle_add_two_pi (x : ℝ) : exp_map_circle (x + 2 * π) = exp_map_circle x :=
 periodic_exp_map_circle x
+
+/-- `exp_map_circle`, applied to a `real.angle`. -/
+noncomputable def exp_map_circle_of_angle (θ : real.angle) : circle :=
```

Comment:
Should this use dot-notation? `real.angle.exp_map` or `.exp_map_circle`? Or maybe `real.angle.cirle`?

Reply:
I don't know whether it should be in the `real.angle` namespace. Another question might be whether it should be a bundled equiv rather than a plain function (a bundled equiv would also prevent use of `θ.exp_map_circle` notation, I think). (It has more structure than just an equiv, but I don't think we have types defined for maps or equivs that send additive to multiplicative monoids, and a version using the types `multiplicative real.angle` or `additive circle` would probably be awkward to use in practice.)

Reply:
For now I've moved it to `real.angle.exp_map_circle` (but not made it an equiv).